### ISSUE TITLE
Refresh token

### ISSUE THREAD

**Issue by sebastianbouckaert** [Viincenttt/MollieApi#255]
The method to refresh the token does not seems to be implemented correctly?
It does does not return an object but a Task ?

**Comment by DianaKoenraadt** [comment]
There might be something missing here. The `TokenRequest` class has a constructor that takes `code` and `redirectUri`. The property `GrantType` and `RefreshToken` cannot be assigned to. (Version 2.1.0.9)

Can the `TokenRequest` class be extended with a constructor to set `grantType` and `refreshToken`?

**Comment by DavidVTurley** [comment]
I agree with Diana Laa. Im not quite sure how to generate a refresh token in the current system.

The TokenRequest class that is fed into the GetAccessToejnAsync function, takes the code generated by the GetAuthorizationUrl in the ConnectClient. That always starts with auth_. That would mean I can never generate a refresh token.

I must be missing something, but I cant find out what.
